packages/terra-table/src/subcomponents/HeaderCell.module.scss

```
@@ -0,0 +1,132 @@
+@import '~terra-mixins';
+
+:local {
+  .header-cell {
+    align-items: flex-end;
+    border: 0;
+    border-right: var(--terra-table-header-cell-border-right, 1px solid #ccc);
+    color: var(--terra-table-header-cell-color, #1c1f21);
+    display: flex;
+    flex: 1 1 0;
+    font-size: var(--terra-table-header-cell-font-size, 1rem);
+    font-weight: var(--terra-table-header-cell-font-weight, bold);
+    justify-content: space-between;
+    line-height: var(--terra-table-header-cell-line-height, 1.143);
+    margin: 0;
+    padding: 0;
+    position: relative;
+    text-align: left;
+    vertical-align: bottom;
+    word-break: break-word;
+
+    &:last-of-type {
+      border-right: 0; // Hide last border
+    }
+  }
+
+  .sort-indicator-asc,
+  .sort-indicator-desc {
+    color: transparent;
+    flex: 0 0 auto;
+    height: var(--terra-table-header-cell-sort-indicator-height, 0.6429rem);
+    margin-bottom: var(--terra-table-header-cell-sort-indicator-margin-bottom, 0.21428rem);
+    padding: 0;
+    width: var(--terra-table-header-cell-sort-indicator-width, 0.6429rem); 
+  }
+
+  .sort-is-active.sort-indicator-asc {
+    background: var(--terra-table-header-cell-desc-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" class="is-bidi"><path fill="#000" d="M24 0L.1 23.5l3.8 3.8L21 10.6V48h6V10.6l17.1 16.7 3.8-3.8z"/></svg>'));
+  }
+
+  .sort-is-active.sort-indicator-desc {
+    background: var(--terra-table-header-cell-sort-asc-background, inline-svg('<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48" class="is-bidi"><path fill="#000" d="M47.9 24.5l-3.8-3.8L27 37.4V0h-6v37.4L3.9 20.7.1 24.5 24 48z"/></svg>'));
+  }
+
+  .is-interactable {
+    cursor: pointer;
+    outline: var(--terra-table-header-cell-outline, none);
+    
+    &:hover {
+      background-color: var(--terra-table-header-cell-hover-background-color, #f1f1f2);
+      background-image: var(--terra-table-header-cell-hover-background-image, linear-gradient(to bottom, #d1d2d3 2%, #e0e1e2 30%, #ebebeb 80%));
+      box-shadow: var(--terra-table-header-cell-hover-box-shadow, 0 4px 4px -2px rgba(134, 138, 140, 0.3), inset 0 0 0 1px #f1f1f2);
```

```suggestion
      box-shadow: var(--terra-table-header-cell-hover-box-shadow, inset 0 0 0 1px #f1f1f2);
```
remove first shadow, leave only:
inset 0 0 0 1px #f1f1f2